website/src/preferences.md

```diff
@@ -1,9 +1,27 @@
 # Preferences
 
 You can see all preferences via the [config](commands/config.md) command and
-change them via the [setup assistant](commands/config-setup.md).
+change them via the [setup assistant](commands/config-setup.md) or manually.
+Configuration data exists on multiple levels:
 
-Git Town can store preferences in two places:
+1. Team-wide configuration settings go into the
+   [configuration file](configuration-file.md). These settings apply to all Git
+   Town users working on the respective repository.
+2. Each developer can configure their preferred Git Town settings for all
+   repositories on their machine using global Git metadata. These settings
+   override (1). For example, if I always want to use the `rebase`
+   [sync-feature-strategy](https://www.git-town.com/preferences/sync-feature-strategy.html)
+   in all my repositories, I would run:
 
-- [configuration file](configuration-file.md)
-- Git metadata: as entries in the local or global Git configuration
+   ```command-summary
+   git config --global git-town.sync-feature-strategy rebase
+   ```
```

Comment:
@stephenwade, this new type of code block doesn't wrap like the others, and it seems to be due to the absence of square brackets. Would there be a different identifier—other than `command-summary`—that could be applied here to make this example Git call wrap similarly to the command summaries? This exceeds my CSS-Fu.

![image](https://github.com/user-attachments/assets/29f158bf-ef4e-4df8-bd24-f8f92ba77ca0)